src/vs/workbench/contrib/terminal/browser/remoteTerminalBackend.ts

```diff
@@ -88,7 +88,7 @@ class RemoteTerminalBackend extends BaseTerminalBackend implements ITerminalBack
 			}
 		});
 
-		const allowedCommands = ['_remoteCLI.openExternal', '_remoteCLI.windowOpen', '_remoteCLI.getSystemStatus', '_remoteCLI.manageExtensions'];
```

Comment:
const allowedCommands = ['_remoteCLI.openExternal', '_remoteCLI.windowOpen', '_remoteCLI.getSystemStatus', '_remoteCLI.manageExtensions', '_remoteCLI.setClipboard'];

Reply:
I cannot see a difference in your comment to what I wrote. Can you describe what you mean?